config/plots/truth/hill.yml

```diff
@@ -0,0 +1,9 @@
+# Plots the distance between the satellites over time
+- type: Plot2D
+  x: truth.t.s
+  y: truth.leader.hill.dr.norm
+
+# Plots the relative velocity of the satellites over time
+- type: Plot2D
+  x: truth.t.s
+  y: truth.leader.hill.dv.norm
```

Comment:
We also probably want to add plots of the x, y, and z hill frame position and velocity components over time? Commonly with this controller the `z` convergence is far slower because matching orbital planes is a little more challenging to control -- adding these plots would make that behavior apparent if rendezvous times were seeming long.